WordPress/UITestsFoundation/Screens/CommentsScreen.swift

```diff
@@ -0,0 +1,63 @@
+import ScreenObject
+import XCTest
+
+public class CommentsScreen: ScreenObject {
+
+    private let replyTextViewGetter: (XCUIApplication) -> XCUIElement = {
+        $0.textViews["ReplyText"]
+    }
+
+    private let backButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.navigationBars.buttons["Reader"]
+    }
+
+    private let replyButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Reply"]
+    }
+
+    var replyTextView: XCUIElement { replyTextViewGetter(app) }
+    var backButton: XCUIElement { backButtonGetter(app) }
+    var replyButton: XCUIElement { replyButtonGetter(app) }
+
+    init(app: XCUIApplication = XCUIApplication()) throws {
+        try super.init(
+            expectedElementGetters: [
+                // swiftlint:skip:next opening_brace
+                { $0.navigationBars["Comments"] },
```

Comment:
So... I've got "inspired" by some other ScreenObject and  initially though it would be ok since this element has no user interactions but your curiosity around that made me rethink it and now I notice at least two flags; 1-Braking getter the pattern and 2-Skipping the linter should have a strong reason behind. So I added a getter in [a19e64b](https://github.com/wordpress-mobile/WordPress-iOS/pull/20432/commits/a19e64ba3355da4451504032220eda12d16bb044).
Thanks for raising the point! 🙇‍♂️